computer_vision/pooling_functions.py

```diff
@@ -115,21 +120,52 @@ def avgpooling(arr: np.ndarray, size: int, stride: int) -> np.ndarray:
     return updated_arr
 
 
-# Main Function
+# Global Average pooling Function
+def global_average_pooling(arr: np.ndarray) -> np.ndarray:
+    """
+    This function performs global average pooling on the input 2D matrix (image).
+    Args:
+        arr: numpy array
+            The input 2D matrix.
+    Returns:
+        numpy array of the global average pooled matrix.
```

Comment:
```suggestion
def global_avg_pooling(arr: np.ndarray) -> np.ndarray:
    """
    Performs global average pooling on the input image matrix.
    Args:
        arr: the input 2D matrix.
    Returns:
        The global average pooled matrix
```
Changed name from "...average..." to "...avg..." for consistency with `avg_pooling`